### PR TITLE
feat: add how to use app in about section

### LINKED ISSUE #632: Add how to use the app in about section
**Describe the feature you'd like**
A how to use app feature like in eventbrite should be there in the about section which would redirect user to support site

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by GOVINDDIXIT**
Fixes #632

Changes: how to use the app is added in about section of setting which will open the support site of the app.

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt [hunk 1]
```diff
@@ -22,6 +22,7 @@ class SettingsFragment : PreferenceFragmentCompat(), PreferenceChangeListener {
     private var email: String? = null
     val EMAIL: String = "EMAIL"
     val FORM_LINK: String = "https://docs.google.com/forms/d/e/1FAIpQLSd7Y1T1xoXeYaAG_b6Tu1YYK-jZssoC5ltmQbkUX0kmDZaKYw/viewform"
+    val SITE_LINK: String ="http://support.eventyay.com"
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt]: This site does not tell how to use the app.

Comment (GOVINDDIXIT) [reply app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt]: Is there any other site? I found this most similar for this purpose having section like event setup, managing orders, payment, ticketing etc.

Comment (simarsingh24) [reply app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt]: for now we dont have a site that tells how to use the app

Comment (GOVINDDIXIT) [reply app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt]: So should i change the title to support site or else. I also see the potential in this site for our required purpose in future

Comment (simarsingh24) [reply app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt]: the site is a general FAQ site

Comment (GOVINDDIXIT) [reply app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt]: Ok @simarsingh24  should I close this PR or change How to use the app to FAQ till we don't get a dedicated site for this purpose.

**Pull request closed without merging**